aws/cluster/variables.tf

```diff
@@ -658,3 +658,9 @@ variable "allowed-unsafe-sysctls" {
   description = "List of sysctls to allow override - allowedUnsafeSysctls (allowed-unsafe-sysctls)"
   default     = []
 }
+
+variable "docker-auth-config" {
+  type        = "string"
+  description = "Base64 encoded ~/.docker/config.json for Docker Hub login"
```

Comment:
If the contents are supposed to be encoded, would it be better to use https://www.terraform.io/docs/configuration/functions/base64encode.html in this module instead of leaving it up to the calling config roots?

Reply:
Well, the input should be somewhere secure like KMS as it includes passwords - we would pass through the correctly encoded and encrypted value to here.